bbb-common-web/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  postgres:
+    image: postgres:latest
```

Comment:
Instead of `latest` we should use fixed tag, `14.1` should be fine for now. The reason is that moving some databases from 13 to 14 brought some systems down. Better avoid such a scenario in the future.